curl_multi_info_read_basic.phpt

```diff
@@ -0,0 +1,58 @@
+--TEST--
+Test function curl_multi_info_read() by calling it with its required argument
+--CREDITS--
```

Comment:
Please add SKIPIF here - look at other CURL tests as an example. Also, please move to ext/curl/tests/